src/extension/app/store/site.js

```diff
@@ -275,6 +281,7 @@ export class SiteStore {
     this.project = project;
     this.devUrl = devUrl;
     this.lang = lang || getLanguage();
+    this.ready = true;
```

Comment:
If you do change `ready` to an `observable` then you would have to set it in a function decorated as an action.

```javascript
@action
  setReady() {
    this.ready = true;
  }
```